Source/Core/Core/PowerPC/Interpreter/Interpreter_LoadStore.cpp

```diff
@@ -782,6 +819,12 @@ void Interpreter::lwarx(UGeckoInstruction inst)
   const u32 address = Helper_Get_EA_X(inst);
   const u32 temp = PowerPC::Read_U32(address);
 
+  if ((address & 0b11) != 0)
+  {
+    GenerateAlignmentException(address);
```

Comment:
Are you sure about the order of Read_U32 and this line? I guess the alignment exceptions has a higher precedence than DSI.

Reply:
This should occur before the read, I'll amend the PR shortly.